Explorer/Assets/DCL/SDKComponents/AvatarModifierArea/Components/AvatarModifierAreaComponent.cs

```diff
@@ -7,13 +7,13 @@ public struct AvatarModifierAreaComponent
     {
         public readonly HashSet<string> ExcludedIds;
 
-        public AvatarModifierAreaComponent(RepeatedField<string> excludedIds)
+        public AvatarModifierAreaComponent(IEnumerable excludedIds)
         {
             ExcludedIds = new HashSet<string>();
             SetExcludedIds(excludedIds);
         }
 
-        public void SetExcludedIds(RepeatedField<string> excludedIds)
+        public void SetExcludedIds(IEnumerable excludedIds)
```

Comment:
IEnumerable < string > ?

Reply:
one minor change and it seems good to me

Reply:
Tackled [here](https://github.com/decentraland/unity-explorer/pull/336/commits/521720a02f3845b5a5f08dfaa2b72439c79c0793)